Source/UI/Controllers/MLCardFormViewController.swift

```diff
@@ -86,6 +87,13 @@ internal extension MLCardFormViewController {
         controller.viewModel.updateWithBuilder(builder)
         return controller
     }
+    
+    func clearFieldsOnFirstAccess() {
```

Comment:
maybe this function could be private? does it has external callers?

Reply:
At this moment only internal.

Reply:
done